provisioning-workflows/IIB/IIBv9/jcl/DeleteMountpointAndBrokerFS.jcl

```diff
@@ -0,0 +1,15 @@
+//MOUNT    EXEC PGM=IKJEFT01
+//SYSTSPRT DD SYSOUT=*
+//SYSTSIN  DD *
+UNMOUNT FILESYSTEM('SYSTEM.HFS.PLEX.VAR.MQSI.${instance-IIB_BROKER_NAME}') +
+IMMEDIATE
+/*
+//S1     EXEC USSBATCH,SU=Y
```

Comment:
USSBATCH is not a standard utility available on z/OS, maybe we should find a more standard approach.